apps/docs/src/app/core/component-docs/object-marker/examples/object-marker-example.component.html

```diff
@@ -1,5 +1,5 @@
-<span fd-object-marker glyph="add-favorite" title="Favorite" aria-label="Favourite Icon" aria-hidden="true"></span>
-<span fd-object-marker glyph="flag" title="Flag" aria-label="Favourite Flag" aria-hidden="true"></span>
-<span fd-object-marker glyph="request" title="Request" aria-label="Favourite Request" aria-hidden="true"></span>
-<span fd-object-marker glyph="user-edit" title="Editable" aria-label="Favourite Editable" aria-hidden="true"></span>
-<span fd-object-marker glyph="private" title="Private" aria-label="Favourite Private" aria-hidden="true"></span>
+<span fd-object-marker glyph="add-favorite" title="Favorite" aria-label="Favorite Icon" aria-hidden="true"></span>
+<span fd-object-marker glyph="flag" title="Flag" aria-label="Favorite Flag" aria-hidden="true"></span>
+<span fd-object-marker glyph="request" title="Request" aria-label="Favorite Request" aria-hidden="true"></span>
+<span fd-object-marker glyph="user-edit" title="Editable" aria-label="Favorite Editable" aria-hidden="true"></span>
+<span fd-object-marker glyph="private" title="Private" aria-label="Favorite Private" aria-hidden="true"></span>
```

Comment:
I think you need to set `aria-hidden=false` or else the screen reader will skip these.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute